### ISSUE TITLE
Tabs

### ISSUE THREAD

**Issue by parsiad** [lars-berger/GlazeWM#233]
Are there any plans to support tabs similar to the tabbed [layout in i3](https://i3wm.org/docs/userguide.html#_changing_the_container_layout)?

**Comment by notblam** [comment]
This is asked pretty often, and I agree something like this would be great.  Occasionally there is some talk about how to support things like this in the form of scriptable layouts or fully automatic layouts.  No one has implemented anything or identified what the best approach would be.  The changes needed would likely be pretty big.

**Comment by asmodeus812** [comment]
That would indeed be fantastic, something that is missing by default in i3 for tabs is being able to pick a tab for example (similarly to how in browsers we can use ctrl + tab number to open / focus the tab)

**Comment by gespau** [comment]
I would also love a tabbed layout. maybe it will come after #604 
If this feature were implemented, I wouldn't even need a maximize feature ...

**Comment by rmanthorpe** [comment]
I would also love to see tabs. It's a feature I make use of a lot in i3wm.

**Comment by khusmann** [comment]
Agreed, this is the only feature in GlazeWM missing for me! For now I'm getting a sort of pseudo-tabbed behavior (for single-screen apps) by fullscreening them, and then [alt-tabbing between them](https://github.com/glzr-io/glazewm/issues/738).

The downside is that fullscreened apps cover the zebar...

**Comment by christopheleroux** [comment]
Hello, 

I'm an unconsented windows user and GlazeWM is my sunlight in the dark =)
I3wm tabbed layout like in GlazeWM would be the 🍒 on the cake .

Thank by advance if you have time to consider this issue (and maybe I would have enough time to learn rust ... in the future )

**Comment by estavrionto** [comment]
need tabbed layout in glazewm pleaase
